src/main/java/org/folio/innreach/domain/service/impl/CirculationServiceImpl.java

```diff
@@ -457,9 +457,6 @@ private void executeDeleteVirtualRecordsWithDelay(TransactionHold hold) {
     var task = new FolioAsyncExecutorWrapper(folioExecutionContext,
             () -> virtualRecordService.deleteVirtualRecords(folioItemId, folioHoldingId,
                     folioInstanceId, folioLoanId));
-
-    if(taskExecutor.schedule(task, new Date(System.currentTimeMillis() + checkOutTimeDuration)).isDone())
```

Comment:
You deleted the scheduling too. So the task will not be scheduled and started.

Reply:
Included that one